README.md

```diff
@@ -92,12 +108,17 @@ Here's a hands-on exercise to illustrate the power of GitOps:
 The beauty of GitOps lies in its ability to seamlessly reflect your git repository's state onto your cluster. Should you decide that the golden glimmer of podinfo doesn't quite suit your taste, reverting is a breeze:
 
 1. Whip open your terminal and run `git log` to retrieve the hash of the commit you wish to undo.
+
+> Connect this with the Gitlens extension, e.g. link directly to Git history? Maybe too fancy for now.
+
 2. Swiftly revert the change with `git revert <COMMIT HASH>` followed by `git push` to update the repository.
 3. Nudge the process along by refreshing the podinfo application within the Argo CD dashboard.
 4. Keep an eye on the transition by running `watch kubectl get pods -n podinfo`. This command lets you observe in real-time as Kubernetes orchestrates the replacement of the pod.
 5. Be greeted once more by the familiar, soothing jade green background of podinfo as the changes take hold.
 
-### Exercise 5: Unterstand GitOps End-to-End
+### Exercise 5: Understand GitOps End-to-End
```

Comment:
mhh I definitely need to make the section more engaging. 
Not sure about the order. I think this should still be located before deploying your own application so that you understand whats going on and then try it yourself when deploying your own application so that it clicks. I noticed that without this chapter users manage to do the tasks without learning anything because they can just follow instructions 1:1.

I think I should move this after game-2048 but still leave it before deploying your own application and before deploying the real world example.

Reply:
I'm not 100% sure if most users are interested in the custom SSH or if they are overwhelmed by it. We must explain GitOps as a concept though. Maybe we can clear up some of the Codespace-specific explanations and leave it generic.